pkg/cloudprovider/gce/gce.go

```
@@ -254,17 +258,34 @@ func (gce *GCECloud) UpdateTCPLoadBalancer(name, region string, hosts []string)
 		Instances: refs,
 	}
 
-	_, err := gce.service.TargetPools.AddInstance(gce.projectID, region, name, req).Do()
+	op, err := gce.service.TargetPools.AddInstance(gce.projectID, region, name, req).Do()
+	gce.waitForRegionOp(op, region)
```

Don't ignore the error return value from waitForRegionOp here